src/test/java/com/github/paweladamski/httpclientmock/HttpClientResponseBuilderTest.java

```diff
@@ -164,6 +167,21 @@ public void should_add_header_to_response() throws IOException {
         assertThat(second.getFirstHeader("tracking").getValue(), equalTo("456"));
     }
 
+    @Test
+    public void should_add_cookie_to_response() throws IOException {
```

Comment:
Name of the test should be `should_add_cookie_to_context`

Reply:
renamed